src/Kiota.Web/Pages/Generate.razor

```diff
@@ -1,71 +1,28 @@
 ﻿@page "/generate"
 @using Kiota.Builder
-@using Kiota.Builder.Configuration
+@using Kiota.Web.Components
+@using Microsoft.Extensions.Localization
 @using System.Linq
-@using System.IO
-@using System.IO.Compression
-@using System.Security.Cryptography
-@using System.Text
-@using System.Text.Json
-@using System.Text.Json.Serialization;
-@using System.Threading
-@using Microsoft.Fast.Components.FluentUI
-@using Microsoft.Kiota.Abstractions.Authentication;
-@inject HttpClient Http
-@inject ILoggerFactory LoggerFactory
-@inject IJSRuntime JSRuntime
 @using System.Globalization
-@using Microsoft.Extensions.Localization
-@inject IStringLocalizer<Generate> Loc
 
+@inject ILoggerFactory LoggerFactory
+@inject IJSRuntime JSRuntime
+@inject IStringLocalizer<SdkGenTranslator> Loc
+@inject KiotaSearcher? KiotaSearcher
 
 <PageTitle>@Loc["PageTitle"]</PageTitle>
```

Comment:
localization for the page title is going to be missing now that everything has been moved to the component. We should add resx for the page.